app/react/components/Widget/Widget.stories.tsx

```diff
@@ -9,7 +11,7 @@ import { WidgetTaskbar } from './WidgetTaskbar';
 interface WidgetProps {
   loading: boolean;
   title: string;
-  icon: string;
+  icon: string | ReactNode;
```

Comment:
`string` is a `ReactNode`, so can replace this with 
```suggestion
  icon: ReactNode;
```

Reply:
updated all occurrences of this